nodes/AsyncFor.py

```diff
@@ -1,3 +1,29 @@
+"""
+AsyncFor astroid node
+
+This node iterates over async code with a for-loop like syntax. Asynchronous
+code doesn't wait for an operation to complete, rather the code executes all
+operations in one go. Only valid in body of an AsyncFunctionDef astroid node.
+
+Attributes:
+    - target  (Node(Name|Tuple|List))
+        - Initial value for the iteration variable and may be any type,
+          not just a number. Equivalent to for loop's i = 0.
+    - iter    (Node)
+        - Holds the item to be looped over.
+    - body    (List[Node])
+        - Contains lists of nodes to execute.
+    - orelse  (List[Node])
```

Comment:
It looks like this attribute could be `None` as well? If so, this should be reflected in the type.